src/vs/workbench/api/common/extHostTypeConverters.ts

```diff
@@ -1023,6 +1023,19 @@ export namespace DocumentHighlight {
 	}
 }
 
+export namespace MultiDocumentHighlight {
+	export function from(multiDocumentHighlight: vscode.MultiDocumentHighlight): languages.MultiDocumentHighlight {
+		return {
+			uri: multiDocumentHighlight.uri,
```

Comment:
we can follow other places where we convert the uri by `Uri.from`

Reply:
Use `URI.revive` instead of `.from`

Reply:
Is using `URI.revive` necessary since the incoming parameter is a URI already rather than UriComponents?

Reply:
In that case not. This is how it usually works

1. Send data from EH to renderer: nothing is required because URI implements toJSON
2. Receiving data inside EH from renderer: use revive on URIs (revive is better than from because it also restores some cached state like the string representation and fsPath)

Reply:
Thanks for the clarity. Will change to follow this pattern and match better alongside the rest of the typeConverters within EH